opensearch_dashboards.json

```diff
@@ -3,8 +3,12 @@
   "version": "3.0.0.0",
   "opensearchDashboardsVersion": "3.0.0",
   "configPath": ["anomaly_detection_dashboards"],
-  "requiredPlugins": ["navigation"],
-  "optionalPlugins": [],
+  "requiredPlugins": ["navigation", 
```

Comment:
nit / unrelated: while you're changing this file, I think we can go ahead and remove `navigation`. It was set when the plugin was initially created before this was understood and it isn't needed.

Reply:
removed, also added visAugmenter which I forgot to add originally here when simplifying the PR